.circleci/config.yml

```diff
@@ -108,13 +117,28 @@ commands:
                 command: circleci-agent step halt
 
 jobs:
-  'Compute Merge Commit':
+  'Initialize Git':
```

Comment:
nit: This name makes it sound like all that's being done is installing the `git` command line tool. How about renaming this to something like "Initialize Repository"?

Reply:
Done.